docs/sdk/web/adapters/coinbase.mdx

```diff
@@ -101,6 +105,22 @@ If you do pass `chainConfig` in an Adapter, it overwrites the `chainConfig` pass
 
 Checkout the [Coinbase Wallet SDK Documentation](https://docs.cloud.coinbase.com/wallet-sdk/docs/initializing#initializing) for these options.
 
+## Change Adapter Settings
+
+You can change the adapter settings by calling the `setAdapterSettings()` function on the adapter instance.
+
```

Comment:
Add all the arguments it takes. An example is not expected in SDK reference.

Table and Interface is needed here. Update this in all adapter documentations.